### PR TITLE
Add optional Rust requirement

### PR DESCRIPTION
**Pull request by ffflorian**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,7 +23,8 @@ Based on [Electron](http://electron.atom.io).
 
 ### Install
 
-Install [Node.js](https://nodejs.org/).
+* Install [Node.js](https://nodejs.org/).
+* If you wish to use [libsodium-neon](https://github.com/wireapp/libsodium-neon/) (encryption speed improvements), also install [Rust](https://rustup.rs).
```
Comment (lipis) [inline README.md]: We are usually using the `-` for lists

**Comment by maximbaz** [comment]
> If you wish to use [libsodium-neon](https://github.com/wireapp/libsodium-neon/) (crypto speed improvements), also install [Rust](https://rustup.rs).

@ConorIA we wish, don't we? 🙂 

Currently it's not used according to the logs I see in the console, but the promised perf improvements are too attractive to avoid 🙂

> Failed loading "libsodium-neon", falling back to "libsodium.js". Error: Cannot find module '/usr/lib/wire-desktop/resources/app/node_modules/libsodium-neon/native'

Time to update PKGBUILDs 😄 
I haven't looked in details yet what exact changes are needed.

**Comment by ffflorian** [comment]
> @ConorIA we wish, don't we? :slightly_smiling_face:

Yes :smile: 

> I haven't looked in details yet what exact changes are needed.

For us the only change was to [install Rust beforehand](https://github.com/wireapp/wire-desktop/blob/master/linux.Jenkinsfile#L27). :slightly_smiling_face:

**Comment by maximbaz** [comment]
I had rust installed for quite a while, what I didn't notice earlier is that libsodium-neon compilation fails every time 🙂 

Details in https://github.com/wireapp/libsodium-neon/issues/15

Still need to modify PKGBUILDs to include rust as (optional) dependency.

**Comment by ConorIA** [comment]
I can add it this afternoon!

**Comment by maximbaz** [comment]
Cool, thanks 🙂 By the way, let me know if you also hit that same compilation failure.

**Comment by ConorIA** [comment]
It's a shame that there is no "optmakedep" category!

**Comment by ConorIA** [comment]
@maximbaz, I've added the rust makedep. Do we need cargo? It is a little hard to test over ssh because I lost most of the build log by the time it finishes.

**Comment by maximbaz** [comment]
Yes, I think cargo is also needed.

**Comment by maximbaz** [comment]
@ConorIA Thinking now, probably it's best to add cargo instead of rust altogether, rust will be installed together with cargo anyway.

**Comment by ffflorian** [comment]
Yes, cargo is needed. `curl https://sh.rustup.rs -sSf | sh -s -- -y` works really good.

**Comment by ConorIA** [comment]
@maximbaz yes, I thought that too.  Will change the optdepend when I get home on a bit.

**Pull request merged**